Explain why we do this (#9)
mui/react-technical-challenge: README.md

# ⚛️ Technical challenge @ MUI

This challenge is part of the hiring process at MUI for some of the Software Engineer positions.
The idea is to make as much progress as possible under a given time constraint (3-4 hours).

## Why are we doing this?

At MUI, because we are a DevTools company, you'll make product decisions. You will flesh out product requirements and turn them into a technical design and implementation.

> This challenge simulates that. 

Author: ```suggestion
This challenge simulates that, it ranges from taking frontend product decisions down to diving into a complex codebase to implement a specific outcome.
```

> This challenge simulates that, it ranges from taking frontend product decisions down to diving into a complex codebase to implement a specific outcome.

Author: ```suggestion
This challenge simulates that, we will review the product decisions you make, the quality of the code, as well as how you approach diving into a complex codebase.
```

> We want to get a glimpse of how you will perform in the role.

Author: Implicit

```suggestion
```

## Context about MUI

In the short term, MUI is on a mission to become the UI toolkit for React. We are unifying the fragmented ecosystem of dependencies into a single set of simple, beautiful, consistent, and accessible React components.

In the long term, MUI is on a mission to make building great React UIs and web applications quicker, simpler, and accessible to more people through a low-code platform. It's about writing fewer code: https://youtu.be/GnO7D5UaDig?t=2451.

> 

Reviewer: *It's about writing fewer lines of code or Its about writing lesser code?

> In the long term, MUI is on a mission to make building great React UIs and web applications quicker, simpler, and accessible to more people through a low-code platform. It's about writing fewer code: https://youtu.be/GnO7D5UaDig?t=2451.

Author: ```suggestion
In the long term, MUI is on a mission to make building great React UIs and web applications quicker, simpler, and accessible to more people through a low-code platform. It's about writing less code: https://youtu.be/GnO7D5UaDig?t=2451.
```
